Big-Spender/readme.md

```diff
@@ -68,39 +68,48 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends where description ilike '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select s.*
+from spends s
+join expense_areas ea on s.expense_area_id = ea.id
+where ea.expense_area ilike 'Better Hospital Food';
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+select to_char(date,'YYYY-MM') as month from spends group by month;
 ```
 
 **Farnoosh:** Thanks, that's really useful. We also need to know the total amount spent on each supplier. Can you help us with that?
 
 **You:** Sure thing. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select
+  supplier_id as supplier,
+  sum(amount) as total_amount
+from spends
+group by supplier;
 ```
 
 **Farnoosh:** Oh, how do I know who these suppliers are? There's only numbers here.
 
 **You:** Whoops! I gave you ids to key the totals, but let me give you names instead.
 
 ```sql
-INSERT YOUR QUERY HERE
+select sp.amount, sup.supplier
```

Comment:
The question is for the *total* amount spent by each supplier. Which operator missing here is needed to get the desired result?